jetson_containers/build.py

```diff
@@ -76,6 +77,14 @@
 print(f"-- PYTHON_VERSION={PYTHON_VERSION}")
 print(f"-- LSB_RELEASE={LSB_RELEASE} ({LSB_CODENAME})")
 
+# cast build args into dictionary
+if args.build_args:
+    try:
+        key_value_pairs = args.build_args.split(',')
+        args.build_args = {pair.split(':')[0]: pair.split(':')[1] for pair in key_value_pairs}
```

Comment:
This line has bug while the value contains colon, for example "http_proxy:http://127.0.0.1:7897". Then `pair.split(':')[1]` has the wrong value "http" instead of "http://127.0.0.1:7897". Passing the maxsplit argument to the split function can fix this bug.